tools/ClassGen/NodeGen.cpp

```diff
@@ -120,6 +120,17 @@ int main(int argc, char **argv) {
                     "Momentum. Similar to Caffe2 SpatialBN, and ONNX "
                     "BatchNormalization operator.");
 
+  BB.newNode("MeanVarNormalization")
+      .addInput("Input")
+      .addInput("Mean")
+      .addInput("Var")
+      .addMember(MemberType::SizeT, "ChannelIdx")
+      .addMember(MemberType::Float, "Momentum")
+      .addResultFromCtorArg("NewMean")
```

Comment:
I think you can write here:
```
       .addResult("Mean.getType()")
       .addResult("Var.getType()")
```
Then you won't have to pass types to the constructor.

Reply:
I don't think this is currently supported for Nodes with multiple outputs. They're both called "Result" and so `getResult()` is defined twice. Could fix this in a future PR.

Reply:
You’ll need to give them a name:
.addResult(“type”, “name”)

Reply:
I didn't realize we supported that! Done.